StepsToCare/CareDashboard.ascx.cs

```diff
@@ -1118,31 +1178,73 @@ public void gList_RowDataBound( object sender, System.Web.UI.WebControls.GridVie
+                                    careNeedFlagStrTooltip += $@"
+                                <div class='assessment-tooltip-item'>
+                                    <span style='color:{randomColor.BackgroundColor};' class='assessment-caretouch'>
+                                        <span class='fa-stack'>
+                                            <i class='fa fa-circle fa-stack-2x'></i>
+                                            <i class='{template.NoteTemplate.Icon} fa-stack-1x' style='color:{randomColor.ForegroundColor};'></i>
+                                        </span>
+                                    </span>
+                                    <span class='assessment-tooltip-value'>
+                                        <span class='assessment-name'>{template.NoteTemplate.Note} Touches Needed</span>
+                                        <span class='assessment-summary'>{touchTooltipTemplate.Replace( "##TouchTitle##", template.NoteTemplate.Note ).Replace( "##TouchCount##", templateNotesToCheckInHours.Count().ToString() ).Replace( "##MinimumTouches##", template.MinimumCareTouches.ToString() ).Replace( "##TouchHours##", template.MinimumCareTouchHours.ToString() )}</span>
+                                    </span>
+                                </div>";
                                     continue;
                                 }
                             }
                         }
-                        else if ( careNeedFollowUpWorkerTouch )
+                        if ( careNeedFollowUpWorkerTouch )
                         {
-                            careNeedFlagStr = "<i class=\"fas fa-flag text-primary mx-2\"  data-toggle=\"tooltip\" title=\"Follow up worker Care Touch Needed!\"></i>";
+                            careNeedFlagStrTooltip += $@"<div class='assessment-tooltip-item'>
+                                <span style='color:#F28E2B;' class='assessment-caretouch'>
+                                    <span class='fa-stack'>
+                                        <i class='fa fa-circle fa-stack-2x'></i>
+                                        <i class='fa fa-user fa-stack-1x text-white'></i>
+                                    </span>
+                                </span>
+                                <span class='assessment-tooltip-value'>
+                                    <span class='assessment-name'>Follow Up Worker</span>
+                                    <span class='assessment-summary'>Touch needed.</span>
+                                </span>
+                            </div>";
+                        }
+                        if ( careNeedFlagStrTooltip.IsNotNullOrWhiteSpace() )
+                        {
+                            careNeedFlagStr = $"<i class=\"fas fa-flag text-danger mx-2\"  data-toggle=\"tooltip\" data-html=\"true\" data-original-title=\"{careNeedFlagStrTooltip}\"></i>";
```

Comment:
This needs to check for quotes in the tooltip to make sure it doesn't break the tooltip since it is customizable.